completion_aggregator/core.py

```diff
@@ -195,13 +217,10 @@ def update_for_aggregator(self, block, affected_aggregators, force):
         last_modified = OLD_DATETIME
 
         if block not in affected_aggregators:
-            try:
-                obj = Aggregator.objects.get(user=self.user, course_key=self.course_key, block_key=block)
-            except Aggregator.DoesNotExist:
-                pass
-            else:
+            obj = self.aggregators.get(block)
```

Comment:
Here's where I save N database queries.  We already have all the aggregator blocks stored in AggregatorUpdater.__init__, so this is redundant.